docs/Adding_a_new_dataset

```diff
@@ -0,0 +1,40 @@
+# Adding a new dataset
+
+This guide is intended for internal collaborators. If you want to add a new dataset to the DFM, but are not a collaborator, please open an issue in this repository or contact us at using the contact form on the [website](https://www.foundationmodels.dk/#join-us).
+
+1) Add a datasheet to the `docs/datasheets` folder, e.g. [`nordjylland_news.md`](https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/main/docs/datasheets/nordjylland_news.md). The datasheet should be named `<dataset_name>.md`. The datasheet should be written in markdown format and should have a front matter following the [Huggingface dataset card template](https://huggingface.co/docs/datasets/en/upload_dataset#create-a-dataset-card). It should have the following attributes:
+   1) a license (if the license is not a standard once allowed by Huggingface, please use "other" and specify the license in the datasheet)
+   2) Languages (e.g. "da" for Danish)
+2) Add a dataset of the same name to the `/danish-foundation-models (193701)/dfm-data/pre-training/` on UCloud. Using the following folder structure:
+
+```
+pre-training
+│
+└── dataset_name
+    │
+    ├── documents
+    │   └── dataset_name.jsonl.gz
```

Comment:
The jsonl files do not need to have the same name as `dataset_name`, and this part of the documentation implies that it is a requirement. Also, it's recommended that the documents are split across multiple files, which makes parallel data processing easier (approximately 1M documents or around 1 GB per file is a sensible tradeoff), so I think it's better if the example shows `part1.jsonl.gz`, `part2.jsonl.gz` etc.

Reply:
@TTTTao725 just to make sure you see this